nixos/modules/services/x11/desktop-managers/pantheon.nix

```diff
@@ -168,12 +168,38 @@ in
       services.udev.packages = [
         pkgs.gnome.gnome-settings-daemon338
       ];
-      systemd.packages = [
-        pkgs.gnome.gnome-settings-daemon338
+      systemd.packages = with pkgs; [
+        gnome.gnome-settings-daemon338
+        gnome.gnome-session
+        pantheon.gala
+        pantheon.elementary-session-settings
       ];
       programs.dconf.enable = true;
       networking.networkmanager.enable = mkDefault true;
 
+      systemd.user.targets."gnome-session-initialized".wants = [
+        "org.gnome.SettingsDaemon.A11ySettings.target"
+        "org.gnome.SettingsDaemon.Color.target"
+        "org.gnome.SettingsDaemon.Datetime.target"
+        "org.gnome.SettingsDaemon.Housekeeping.target"
+        "org.gnome.SettingsDaemon.Keyboard.target"
+        "org.gnome.SettingsDaemon.MediaKeys.target"
+        "org.gnome.SettingsDaemon.Power.target"
+        "org.gnome.SettingsDaemon.PrintNotifications.target"
+        "org.gnome.SettingsDaemon.Rfkill.target"
+        "org.gnome.SettingsDaemon.ScreensaverProxy.target"
+        "org.gnome.SettingsDaemon.Sharing.target"
+        "org.gnome.SettingsDaemon.Smartcard.target"
+        "org.gnome.SettingsDaemon.Sound.target"
+        "org.gnome.SettingsDaemon.UsbProtection.target"
+        "org.gnome.SettingsDaemon.Wacom.target"
+        "org.gnome.SettingsDaemon.Wwan.target"
+      ];
+
+      systemd.user.targets."gnome-session-x11-services".wants = [
+        "org.gnome.SettingsDaemon.XSettings.target"
+      ];
+
```

Comment:
Not sure if we should do this here...? But this should fix the media key issue...